objectivec/GPBDictionary.h

```diff
@@ -60,55 +60,6 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, readonly) NSUInteger count;
 
 /**
- * @return A newly instanced and empty dictionary.
- **/
-+ (instancetype)dictionary;
```

Comment:
Doing some simple searching, this one seemed somewhat common.  That's why I wasn't sure.